src/components/fieldset/address-group.njk

```diff
@@ -17,18 +19,18 @@ layout: layout-example.njk
   </label>
   <input class="govuk-c-input" id="address-line-2" name="address-line-2" type="text">
 
-  <label class="govuk-c-label" for="address-town">
+  <label class="govuk-c-label govuk-!-width-two-thirds" for="address-town">
     Town or city
   </label>
-  <input class="govuk-c-input" id="address-town" name="address-town" type="text">
+  <input class="govuk-c-input govuk-!-width-two-thirds" id="address-town" name="address-town" type="text">
 
-  <label class="govuk-c-label" for="address-county">
+  <label class="govuk-c-label govuk-!-width-two-thirds" for="address-county">
     County
   </label>
-  <input class="govuk-c-input" id="address-county" name="address-county" type="text">
+  <input class="govuk-c-input govuk-!-width-two-thirds" id="address-county" name="address-county" type="text">
 
-  <label class="govuk-c-label" for="address-postcode">
+  <label class="govuk-c-label govuk-!-width-one-third" for="address-postcode">
```

Comment:
Was thinking about this after you mentioned it the other day - if this happened to say e.g. "Postcode / Zip Code" would we really want it wrapping onto a new line to match the width of the input? I can't visualise it, but I feel like allowing the labels the full width of the form would be the preferred behaviour in that instance. Might not be the case for longer inputs, but not sure.

Reply:
I think this would just be a choice for the designer to make. So it needing to be added to both the label and the input gives you more options than you had previously...

Reply:
What do you think the default should be? If designers are just copying this example, and then changing labels to suit their service, do we think the widths should be there by default on labels, or something they _can add_ if they want to? Because I think there will be some who assume it _has_ to be there, and we end up with weird wrapped labels.

Reply:
This is a fair point, if anything they should wrap to the size of the widest form input in the group, not the input they're the label for. I'll remove all together.